.gitignore

```diff
@@ -1,2 +1,4 @@
 # file with secrets
 secrets.yaml
+# the inventory
+inventory.yaml
```

Comment:
It may be good to have an example/template for the inventory file.

Reply:
Sample added, not much to specify there anyway